src/parlant/adapters/nlp/openai.py

```diff
@@ -363,7 +363,7 @@ def __init__(
     async def get_schematic_generator(self, t: type[T]) -> OpenAISchematicGenerator[T]:
         if t == ToolCallInferenceSchema:
             return FallbackSchematicGenerator(
-                GPT_4o_Mini[t](self._logger),  # type: ignore
+                # GPT_4o_Mini[t](self._logger),  # type: ignore
```

Comment:
Don't leave this commented out; remove it. Remove `Fallback` in general here, just return `GPT_4o`.